packages/admin/admin/src/form/FinalFormFileSelect.tsx

```diff
@@ -0,0 +1,273 @@
+                        {isDragReject && <div className={classes.droppableAreaError}>{infoIcon}</div>}
+                        <Typography variant="body2" className={classes.droppableAreaCaption}>
+                            <FormattedMessage id="comet.finalFormFileSelect.dropfiles" defaultMessage="Drop files here to upload" />
+                        </Typography>
+                    </div>
+                )}
+                {!disableSelectFileButton && (
+                    <Button disabled={disabled} variant="contained" color="secondary" startIcon={selectIcon}>
+                        <FormattedMessage id="comet.finalFormFileSelect.selectfile" defaultMessage="Select file" />
+                    </Button>
+                )}
+            </div>
+            {acceptedFiles.length > 0 && (
+                <div className={classes.fileList}>
+                    {acceptedFiles.map((file, index) => (
+                        <div key={index} className={classes.fileListItem}>
+                            <Tooltip trigger="hover" title={file.name}>
+                                <div className={classes.fileListText}>{file.name}</div>
+                            </Tooltip>
+                            <div className={classes.fileListItemInfos}>
+                                <Chip label={<PrettyBytes value={file.size} />} />
+                                <IconButton onClick={removeFile(file)}>{deleteIcon}</IconButton>
+                            </div>
+                        </div>
+                    ))}
+                </div>
+            )}
+            {fileRejections.length > 0 && <div className={classes.fileList}>{rejectedFiles}</div>}
+            {(fileRejections.length > 0 || isDragReject) && (
+                <div className={classes.errorMessage}>
+                    {infoIcon}
+                    <FormattedMessage id="comet.finalFormFileSelect.errors.unknownError" defaultMessage="Something went wrong." />
+                </div>
+            )}
+            <FormHelperText sx={{ margin: 0 }}>
+                <FormattedMessage
+                    id="comet.finalFormFileSelect.maximumFileSize"
+                    defaultMessage="Maximum file size {fileSize}"
+                    values={{
+                        fileSize: <PrettyBytes value={maxSize} />,
+                    }}
+                />
+            </FormHelperText>
+        </div>
+    );
+};
+
+export const FinalFormFileSelect = withStyles(styles, { name: "CometAdminFinalFormFileSelect" })(FinalFormFileSelectComponent);
+
+declare module "@mui/material/styles" {
+    interface ComponentNameToClassKey {
+        CometAdminFinalFormFileSelect: FinalFormFileSelectClassKey;
+    }
+
+    interface ComponentsPropsList {
+        CometAdminFinalFormFileSelect: FinalFormFileSelectProps;
+    }
+
+    interface Components {
+        CometAdminFinalFormFileSelect?: {
+            defaultProps?: ComponentsPropsList["CometAdminFinalFormFileSelect"];
```

Comment:
There should be no required props, when overriding the default values. 

```suggestion
            defaultProps?: Partial<ComponentsPropsList["CometAdminFinalFormFileSelect"]>;
```